tests/react_parity/test_fixtures_and_exporter.py

```
@@ -0,0 +1,69 @@
+
+
+def test_reference_exporter_is_byte_stable(tmp_path: Path) -> None:
+    payload = tmp_path / "payload.json"
+    payload.write_text(
+        json.dumps({"b": 2, "a": [3, 1], "z": float("nan")}, allow_nan=True),
+        encoding="utf-8",
+    )
+    outs = []
+    for i in range(3):
+        out = tmp_path / f"ref{i}.json"
+        subprocess.check_call(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)